import Foundation
import Kotlin

printHelloWorld()

fatalError("The program finished without explicitly calling `exit(0)`. This is interpreted as an error to prevent tests from accidentally passing via unexpected execution path.")
